style/ruby/.rubocop.yml

```diff
@@ -0,0 +1,108 @@
+AllCops:
+  TargetRubyVersion: 2.3
+  # Cop names are not displayed in offense messages by default. We find it useful to include this information so we can
+  # use it to investigate what the fix may be.
+  DisplayCopNames: true
+  # Style guide URLs are not displayed in offense messages by default. Again we find it useful to go straight to the
+  # documentation for a rule when investigating what the fix may be.
+  DisplayStyleGuide: true
+  Include:
+    - "**/*.gemspec"
+    - "**/*.rake"
+    - "**/*.rb"
+    - "**/Gemfile"
+    - "**/Rakefile"
+    - "**/bin/*"
+    - "**/config.ru"
+  Exclude:
+    - "config/**/*"
+    - "script/**/*"
+    - !ruby/regexp /old_and_unused\.rb$/
+    # Rubocop is accidentally linting HTML ERB files in Atom Editor https://github.com/AtomLinter/linter-rubocop/issues/48
+    - "**/*.html.erb"
```

Comment:
Actually, seems like this is fixed now, so we could remove this?

https://github.com/AtomLinter/linter-rubocop/issues/48